app/controllers/expenses_additional_sources_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ExpensesAdditionalSourcesController < StandardStepsController
+  private
+
```

Comment:
yeah, we've been testing this at the controller level so far. Example: https://github.com/codeforamerica/michigan-benefits/blob/13c20027a88425746d695df75abee0040a158da2/spec/controllers/household_add_member_controller_spec.rb#L14